packages/api-headless-cms/__tests__/pluralizeContentModel.test.js

```diff
@@ -0,0 +1,29 @@
+import useContentHandler from "./utils/useContentHandler";
+import mocks from "./mocks/pluralizeContentModel";
+import { createContentModelGroup, createEnvironment } from "@webiny/api-headless-cms/testing";
+
+describe("Pluralize Content Model Test", () => {
+    const { database, environment, invoke } = useContentHandler();
+
+    const initial = {};
+
+    beforeAll(async () => {
+        // Let's create a basic environment and a content model group.
+        initial.environment = await createEnvironment({ database });
+        initial.contentModelGroup = await createContentModelGroup({ database });
+    });
+
+    it("should be able to create a content model with a single capitalized letter and not have the pluralized version be capitalized", async () => {
+        const { content, createContentModel } = environment(initial.environment.id);
+
+        const contentModel = await createContentModel(
+            mocks.contentModel({ contentModelGroupId: initial.contentModelGroup.id })
+        );
+
+        const ps = await content("p");
+
+        const createdP = await ps.create(mocks.createP);
```

Comment:
Let's try this, instead of just calling `ps.list()`, write the query yourself. So `listPs { ... }`.

For example, the following tests are defining GQL queries inline, and executing them using the `invoke` function:
packages/api-headless-cms/__tests__/accessTokens.test.js:51